test/test-pages/gcba/sdk-controller.ts

```diff
@@ -0,0 +1,307 @@
+/* global MediaStream */
+import GenesysCloudWebrtSdk, { ConversationUpdate, IExtendedMediaSession, ISdkConfig, ISdkConversationUpdateEvent, IStoredConversationState, SdkError, SessionTypes } from '../../../src';
+import { getInputValue, writeToLog } from './utils';
+
+let webrtcSdk: GenesysCloudWebrtSdk;
+let pendingSessions = [];
+
+let conversationUpdatesToRender: {[conversationId: string]: IExtendedMediaSession} = { };
```

Comment:
Codacy has a fix for the issue: ['conversationUpdatesToRender' is never reassigned. Use 'const' instead.](https://app.codacy.com/gh/MyPureCloud/genesys-cloud-webrtc-sdk/pullRequest?prid=9580329)
```suggestion
const conversationUpdatesToRender: {[conversationId: string]: IExtendedMediaSession} = { };
```